rules/windows/discovery_peripheral_device.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/11/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of the fsutil Windows utility to gather information about attached peripheral devices and components
+connected to a computer system.
+connected to a computer system.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Peripheral Device Discovery"
+risk_score = 21
+rule_id = "0c7ca5c2-728d-4ad9-b1c5-bbba83ecb1f4"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+(process.name : "fsutil.exe" or process.pe.original_file_name : "fsutil.exe") and process.args : "fsinfo" and process.args : "drives"
+'''
```

It is better to always leave original file name checks case sensitive (of course verifying the actual case)

```suggestion
query = '''
process where event.type in ("start", "process_started") and
  (process.name : "fsutil.exe" or process.pe.original_file_name == "fsutil.exe") and 
  process.args : "fsinfo" and process.args : "drives"
'''
```